openslides/motions/serializers.py

```diff
@@ -488,6 +499,14 @@ def create(self, validated_data):
         motion.supporters.add(*validated_data.get("supporters", []))
         motion.attachments.add(*validated_data.get("attachments", []))
         motion.tags.add(*validated_data.get("tags", []))
+
```

Comment:
When does `validate` get called? If it gets called on every creation of the serializer (=creating, updating, ...) unnecessary autoupdates are send here. I guess the code in `create()` and `update()` should be sufficient.

Reply:
This is the create method ;)

Reply:
fuu git diff view ;) Sorry..